### PR TITLE
additional quota in vsphere 8

### PR DESCRIPTION
**Pull request by jcpowermac**

### CODE REVIEW

File: core-services/prow/02_config/generate-boskos.py [hunk 1]
```diff
@@ -308,6 +308,8 @@
 
 for i in range(200,204):
     CONFIG['vsphere-8-quota-slice']['ci-segment-{}'.format(i)] = 1
+for i in range(205,214):
```
Comment (smg247) [inline core-services/prow/02_config/generate-boskos.py:311]: Why not just extend the above range?

Comment (jcpowermac) [reply core-services/prow/02_config/generate-boskos.py:311]: there is no 204

**Pull request merged**